### DISCUSSION TITLE
Global Search disappeared on nova 4

### DISCUSSION THREAD

**Discussion by WillyReyno** [laravel/nova-issues#4021] [Q&A] [answered]
Hello, 

I'm not sure if I should open an issue about it because it may be related to my project's configuration, but I noticed that my global search bar has disappeared from Nova (v4.2.2)

![Capture d’écran 2022-04-17 à 19 01 50](https://user-images.githubusercontent.com/5576482/163724827-46edc027-0a7b-459f-af19-42bd9d520831.png)

Am I the only one experiencing this? Any idea where that could come from?

**Comment by wdelfuego** [comment]
The same happened to me, I didn't see it happen and I didn't yet have time to investigate so I don't know anything more yet, but my project is in the same state.

**Comment by gmoigneu** [comment]
Same here after the update to 4.x. I'll look into it.

**Comment by crynobone** [comment]
See #4022

**Comment by crynobone** [comment, marked as answer]
Fixed in 4.2.4: https://nova.laravel.com/releases/4.2.4
